### ISSUE TITLE
The pretraining and finetuning time 

### ISSUE THREAD

**Issue by 1024AILab** [hgaurav2k/hop#7]
Hello, I am very curious about how long the pretraining will take? I run the finetuning on two 4090 with 10w epoch, which takes almost three days. What types of GPU do you use?

The 10w epoch finetuning results look good !

https://github.com/user-attachments/assets/50c7c427-d35e-4aaa-8775-3b1c895486b6

**Comment by 1024AILab** [comment]
I double the mini_batchsize and num_envs. Will it shorten the finetuning time? Thank you~

**Comment by 1024AILab** [comment]
These are the visualization results. Were the results correct?

![reward_analysis_AllegroXarmGrasping_Finetuned](https://github.com/user-attachments/assets/1e5dda96-488d-4702-9ee0-26f6e6561c58)
![success_analysis_AllegroXarmGrasping_Finetuned](https://github.com/user-attachments/assets/766e17c7-0785-4677-8424-0ea668e4947a)

**Comment by hgaurav2k** [comment]
The finetuning (~600M agent steps with total batch size (number of agents) 4096) takes us less than 12 hours with 8 A5000s. For pretraining, any checkpoint beyond 50K gradient steps is sufficient for the finetuning stage.

**Comment by ZainZh** [comment]
It takes me almost four days with 4096 agents on one single 4090 GPU to get your reward (around 1800).
